setuptools/py27compat.py

```diff
@@ -4,7 +4,7 @@
 
 import platform
 
-import six
+from setuptools.extern import six
```

Comment:
@fwdevmobile It doesn't mean it *must* have six. But it makes sense for it to use it when available, to - even if only for a tiny bit - increase readability and reduce technical debt. It's unfortunate that there was a mistake in it, but generally, it - IMHO - is a slight improvement.